devlib/instrument/gem5power.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, target, power_sites):
             self.add_channel(field, 'power')
         self.target.gem5stats.book_roi(self.roi_label)
         self.sample_period_ns = 10000000
+        self.sample_rate_hz = 1 / (self.sample_period_ns * 10**-9)
```

Comment:
That's a good point thanks, I've removed that so it will just rely on the timestamps reported. 
On a similar note, to conform with the new `DerivedMeasurements` API, to allow them to be detected the time channel needs to be renamed from "sim_seconds" to "timestamp", is this going to cause many problems?